catalog/app/containers/UriResolver/UriResolver.js

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react'
+import { useHistory, Redirect } from 'react-router-dom'
+import * as M from '@material-ui/core'
+
+import Layout from 'components/Layout'
+import * as NamedRoutes from 'utils/NamedRoutes'
+import * as PackageUri from 'utils/PackageUri'
+
+const useStyles = M.makeStyles((t) => ({
+  container: {
+    paddingTop: t.spacing(6),
+    paddingBottom: t.spacing(6),
+    maxWidth: '656px !important',
```

Comment:
You can avoid `!important` by utilizing `maxWidth` attribute:

```jsx
<M.Container maxWidth="sm" />
```

Reply:
nope, because i need specific custom value

Reply:
Why?

Reply:
because the placeholder looks best at this width :)

Reply:
In this case I would prefer code clarity over UI look